experimental/origin-dapp2/src/pages/_ListingBadge.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
```

Comment:
Lets put this in `src/components` as it'll be used in multiple pages. You can then `import ListingBadge from 'components/ListingBadge'` from any other page